packages/info/src/index.ts

```diff
@@ -61,6 +61,6 @@ export default async function info(...args): Promise<string[]> {
     output = output.replace(/npmGlobalPackages/g, 'Global Packages');
 
     const finalOutput = output;
-    process.stdout.write(finalOutput + '\n');
+    console.log(finalOutput + '\n');
```

Comment:
Let's use `logger.log` here, we should avoid using `process`/`console` directly in code

Reply:
https://github.com/webpack/webpack-cli/blob/b071623ae67a9f9528b02e07376044d851ad378a/test/info/info-output.test.js#L18-L33

`webpack-cli info --output=json` would produce the following output:-
```bash
[webpack-cli] {
  "System": {
    "OS": "macOS 10.15.6",
    "CPU": "(8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz",
    "Memory": "405.52 MB / 8.00 GB"
  },
  "Binaries": {
    "Node": {
      "version": "13.11.0",
      "path": "/usr/local/bin/node"
    },
    "Yarn": {
      "version": "1.22.4",
      "path": "/usr/local/bin/yarn"
    },
    "npm": {
      "version": "6.14.7",
      "path": "/usr/local/bin/npm"
    }
  },
  "Browsers": {
    "Brave Browser": {
      "version": "84.1.12.112"
    },
    "Safari": {
      "version": "13.1.2"
    }
  },
  "Packages": {
    "webpack": {
      "installed": "5.0.0-beta.23",
      "wanted": "^4.44.1"
    }
  },
  "Global Packages": {
    "webpack-cli": "4.0.0-beta.8"
  }
}
```

It fails to recognize `stdout` in the `JSON` format.

Reply:
We can implement `logger.raw('test',)` for this case